docs/documentation/updating-the-kit.md

```diff
@@ -1,129 +1,44 @@
-# Updating the kit
+# Updating to the latest version
 
-**Important note**
+1. Make a backup copy of your prototype folder.
 
-If you have made any changes outside the `app` folder, this process will destroy those changes. We will try and improve the update process to avoid this, but in the meantime you will need to make a note of your changes outside `app`, and add them back after updating.
+2. In your prototype, delete everything except the `app` and `.git` folders.
 
-## Steps
+   You may not be able to see the `.git` folder because files and folders that start with `.` are hidden by default.
 
-[Download the latest Prototype Kit](/docs/download) and unzip it.
+3. [Download the zip file of the latest version of the Prototype Kit](/docs/download).
+  
+4. Unzip the zip file.
+ 
+5. Copy all the folders except the `app` folder to your prototype.
 
-In your project, delete everything apart from the `app` and `.git` folder.
+6. Copy the `app/config.js` file to the `app` folder of your prototype.
```

Comment:
Is it worth clarifying that they're copying `app/config.js` from the unzipped folder?

Reply:
Ah yes, good call. I've added in the suggestion above.